common/utils_win.cc

```diff
@@ -74,11 +106,13 @@ DWORD CreatePipe(
   *handle = CreateNamedPipeA(name.c_str(), mode,
     PIPE_TYPE_MESSAGE | PIPE_READMODE_MESSAGE | PIPE_WAIT |
     PIPE_REJECT_REMOTE_CLIENTS, PIPE_UNLIMITED_INSTANCES, kBufferSize,
-    kBufferSize, 0, /*securityAttr=*/nullptr);
+    kBufferSize, 0, &sa);
   if (*handle == INVALID_HANDLE_VALUE) {
     err = GetLastError();
   }
 
+  LocalFree(sa.lpSecurityDescriptor);
```

Comment:
Freeing this here implies that the created pipe no longer needs access to `sa`.
Could you add a comment detailing that, or correct me if I'm reading this wrong?

Reply:
Correct no longer needed.  Added comment.

Reply:
Correct no longer needed.  Added comment.